src/backend/common/models/media.py

```diff
@@ -281,3 +281,11 @@ def image_direct_url_sm(self) -> str:
     def avatar_image_source(self) -> str:
         image = json.loads(self.details_json)
         return "data:image/png;base64, {}".format(image["base64Image"])
+
+    @property
+    def team_number(self) -> TeamNumber:
+        """
+        Returns the associated team number for a piece of media.
+        """
+
+        return TeamNumber(self.references[0].id()[3:])
```

Comment:
Is `self.references` guaranteed to be non-empty? I don't know much about this model (cc @fangeugene)

Reply:
`Media.references` is not guaranteed to contain references to a `Team` object. It can also be `Event`. But it is guaranteed to be not null. However, `references` being not null and a reference to a `Team` is loosely guaranteed if `media_type_enum` is an avatar.

@cjdenio I see how this is potentially cleaner, but doing `avatar.references[0].id()[3:]` in the template is safer for now. Maybe in the future we can build better typing around all of this.